staging/src/k8s.io/legacy-cloud-providers/azure/clients/storageaccountclient/azure_storageaccountclient.go

```diff
@@ -196,7 +196,7 @@ func (c *Client) listStorageAccountKeys(ctx context.Context, resourceGroupName s
 		azure.WithErrorUnlessStatusCode(http.StatusOK),
 		autorest.ByUnmarshallingJSON(&result))
 	if err != nil {
-		klog.V(5).Infof("Received error in %s: resourceID: %s, error: %s", "storageaccount.listkeys.respond", resourceID, err)
+		klog.V(5).InfoS("Received error in storageaccount.listkeys.respond", "resourceID", resourceID, "error", err)
```

Comment:
this is changing the format of original logging, what's the benefit?

Reply:
```suggestion
		klog.V(5).InfoS("Received error in storageaccount.listkeys.respond", "resourceID", resourceID, "err", err)
```